### DISCUSSION TITLE
如何让bot接收自己的命令？

### DISCUSSION THREAD

**Discussion by j1nse** [nonebot/discussions#120] [问答] [answered]
我用on_command()无法接收到bot自己说话的时候发出的指令，我看后台是[message_sent]

只有[message.group.normal] 时，on_command()才起作用

**Comment by yanyongyu** [comment, marked as answer]
nb2的话 你可以自己写一个rule来匹配message sent事件，command目前只会匹配message事件，如果有需要可以提个issue
